locallibrary/catalog/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
```

Comment:
sắp xếp lại phần import module/package: django -> python -> user-defined 
VD:
```suggestion
from django.db import models
from django.urls import reverse
import uuid
```